qiita_pet/uimodules/prep_template_tab.py

```diff
@@ -218,7 +219,19 @@ def render(self, study, prep_template, full_access, ena_terms,
                         no_preprocess_msg = (
                             "Preprocessing disabled due to missing columns in "
                             "the prep template: %s" % ', '.join(missing_cols))
-        preprocessing_status = 'TODO: plugin'
+
+            # Check the processing status
+            jobs = raw_data.jobs(status='running')
```

Comment:
Got it thanks for the explanation!

On (Dec-04-15|13:47), Jose Navas wrote:
> > @@ -218,7 +219,19 @@ def render(self, study, prep_template, full_access, ena_terms,
> >                          no_preprocess_msg = (
> >                              "Preprocessing disabled due to missing columns in "
> >                              "the prep template: %s" % ', '.join(missing_cols))
> > -        preprocessing_status = 'TODO: plugin'
> > +
> > +            # Check the processing status
> > +            jobs = raw_data.jobs(status='running')
> 
> The reason why it is here is because the artifact (and qiita db) supports multiple processing of an artifact in parallel. However, the interface was designed to be processing only once at a time. I have the feeling that this is a limitation that is going to go away once we re-implement the study interface, so this is just a hack to maintain the same interface.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1558/files#r46736352